### PR TITLE
feat: add embed image for twitter preview

### LINKED ISSUE #218: Show embed image for squeeth site
**Please describe the problem you're having.**
Right now if you link the squeeth site on twitter or elsewhere there is no embed image. We should add an embed image for the site

**Screenshots**
No embed image with squeeth site 
Screen Shot 2022-03-06 at 11 25 21 PM
https://twitter.com/opyn_/status/1496248909085261824?s=20&t=I82wILz4J8kF9a0dheHKtA

Embed image with medium
Screen Shot 2022-03-06 at 11 28 18 PM
https://twitter.com/opyn_/status/1498836927788859394?s=20&t=XFkHvQH2BgaYXUeuyCUTeQ

### PR DESCRIPTION
**Pull request by rootial**
# Task:

## Description
Add embed image to head section of page to enable preview when squeeth is shared in twitter.

> A URL to a unique image representing the content of the page. You should not use a generic image such as your website logo, author photo, or other image that spans multiple pages. Images for this Card support an aspect ratio of 2:1 with minimum dimensions of 300x157 or maximum of 4096x4096 pixels. Images must be less than 5MB in size. JPG, PNG, WEBP and GIF formats are supported. Only the first frame of an animated GIF will be used. SVG is not supported.

Fixes #218 (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Document update

## How Has This Been Tested

Please describe how to test to verify the changes. Provide instructions so we can reproduce.

## Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] Added video recordings if it is a UI change

### CODE REVIEW

**Comment by rootial** [comment]
@alexisgauba hi, can you enable vercel to deploy the pr, then i can test the twitter preview. thanks~

**Review by KMKoushik — APPROVED** [review]
LGTM

**Comment by KMKoushik** [comment]
Screenshot 2022-03-23 at 7 06 19 PM

I can't see the image in Twitter.

**Comment by rootial** [comment]
twitter requires embedded image size between 300x157 or maximum of 4096x4096 pixels and the squeeth logo in project can't qualify for it. Do we have a specific png image for this ? @alexisgauba @KMKoushik

**Comment by KMKoushik** [comment]
> twitter requires embedded image size minium of 300x157 or maximum of 4096x4096 pixels and the squeeth logo in project is 8504x3695 and can't qualify for it. Do we have a specific png image for this ? @alexisgauba @KMKoushik

I will try to get it

**Comment by KMKoushik** [comment]
@rootial here you go
SqueethLogo

File: packages/frontend/pages/_app.tsx [hunk 1]
```diff
@@ -80,6 +80,13 @@ const TradeApp = ({ Component, pageProps }: any) => {
           name="description"
           content="Squeeth is a new financial primitive in DeFi that gives traders exposure to ETH²"
         />
+        <meta name="twitter:card" content="summary_large_image" />
+        <meta name="twitter:title" content="Squeeth" />
+        <meta
+          name="twitter:description"
+          content="Squeeth is a new financial primitive in DeFi that gives traders exposure to ETH²"
+        />
+        <meta name="twitter:image" content="https://squeeth.opyn.co/images/SqueethLogoMedium.png" />
```
Comment (rootial) [inline packages/frontend/pages/_app.tsx:89]: we need this file located in prod firstly before we can test it.

**Comment by rootial** [comment]
add new logo in this pr, please take a look. @KMKoushik 

https://github.com/opynfinance/squeeth-monorepo/pull/264

**Comment by KMKoushik** [comment]
@rootial Thanks merged the logo PR

**Comment by rootial** [comment]
Can you trigger the vercel deploy then i can preview the change? @KMKoushik

**Pull request merged**